xml_inscriptions/bib_only/MA.Bos.MFA.G.01.7479.xml

```diff
@@ -66,14 +131,30 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-06-22" who="Jean Wanlass">expansion of bib only stub</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc>photo: MFA 01.7479</desc>
```

Comment:
Typically we don't use this caption unless it's for things like details, or images that are not our own; you can delete this.